File: Praktikum1/PRAK104-2010817210023-MuhammadAuliaAkbar.py
hargasepatuA = int(input("Masukkan Harga Sepatu A: "))
hargasepatuB = int(input("Masukkan Harga Sepatu B: "))

diskon13 = 13
diskon21 = 21

diskonsepatuA13persen = hargasepatuA - (hargasepatuA * (diskon13 / 100))
diskonsepatuB21persen = hargasepatuB - (hargasepatuB * (diskon21 / 100))
print(f"Harga Sepatu A adalah {int(hargasepatuA)}")
print(f"Harga Sepatu B Adalah {int(hargasepatuB)}")
print(f"Sepatu A Mendapat Diskon {int(diskon13)} Sehingga Harganya Menjadi {int(diskonsepatuA13persen)}")
print(f"Sepatu B Mendapat Diskon {int(diskon21)} Sehingga Harganya Menjadi {int(diskonsepatuB21persen)}")

File: Praktikum1/PRAK104-2010817210023-MuhammadAuliaAkbar.c
#include <stdio.h>

int main()
{
    float hargasepatuA, hargasepatuB;
    float diskon13 = 13;
    float diskon21 = 21;
    printf("Masukan Harga Sepatu A: ");
    scanf("%f", &hargasepatuA);
    printf("Masukan Harga Sepatu B: ");
    scanf("%f", &hargasepatuB);

    float diskonsepatuA13persen = hargasepatuA - (hargasepatuA * (diskon13 / 100));
    float diskonsepatuB21persen = hargasepatuB - (hargasepatuB * (diskon21 / 100));
    printf("Sepatu A Mendapat Diskon %.0f Sehingga Harganya Menjadi %.0f\n", diskon13, diskonsepatuA13persen);
    printf("Sepatu B Mendapat Diskon %.0f Sehingga Harganya Menjadi %.0f", diskon21, diskonsepatuB21persen);
    return 0;
}